src/crab/splitdbm/adapt_sgraph.hpp

```
@@ -377,28 +358,28 @@ class AdaptGraph final {
     }
 
     [[nodiscard]]
-    bool elem(VertId s, VertId d) const {
+    bool elem(const VertId s, const VertId d) const {
         return _succs[s].contains(d);
     }
 
-    const Weight& edge_val(VertId s, VertId d) const { return _ws[*_succs[s].lookup(d)]; }
+    const Weight& edge_val(const VertId s, const VertId d) const { return _ws[*_succs[s].lookup(d)]; }
```

Clang (14.0.6)

[warning] function 'edge_val' should be marked [[nodiscard]]

(modernize-use-nodiscard)